addon/utils/flags.js

```diff
@@ -0,0 +1,5 @@
+function getFlag(str) {
```

Comment:
i think this file can be dropped

Reply:
Agree. We were initially looking to use this in a few places, but it turns out we didn't need to reuse the functionality.

Reply:
So to be clear, I need the constant to be shared, but the function can go. I use the constant in a few places, and need the symbol to be consistent.

Reply:
Removed the `getFlag` function and just inlined the exported const.